staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -79,3 +87,60 @@ func newETCD3Storage(c storagebackend.Config) (storage.Interface, DestroyFunc, e
 	}
 	return etcd3.NewWithNoQuorumRead(client, c.Codec, c.Prefix, transformer, c.Paging), destroyFunc, nil
 }
+
+// loggerT is the default logger that delegates to glog and used by grpclog
+type loggerT struct{}
+
+var _ clientv3.Logger = &loggerT{}
```

Comment:
Why is this line necessary ?

Reply:
to make sure `loggerT`  implements `clientv3.Logger`. If we missed one method, it will fail in compilation.